validator-frontend/screens/ScanScreen.js

```diff
@@ -33,7 +52,109 @@ export default class ScanScreen extends React.Component {
+      '   BAcTCUFtc3RlcmRhbTEWMBQGA1UEChMNQ29tbW9uIEdyb3VuZDEMMAoGA1UECxMD  '  +
+      '   TkxYMB4XDTE5MDExMDEyMTIwMFoXDTIwMDExMDEyMTIwMFowYzELMAkGA1UEBhMC  '  +
+      '   TkwxFTATBgNVBAgTDFp1aWQtSG9sbGFuZDERMA8GA1UEBxMIRGVuIEhhYWcxDTAL  '  +
+      '   BgNVBAoTBElDVFUxGzAZBgNVBAsTElN5c3RlbSBUZXN0IFRyYXZpczCCAiIwDQYJ  '  +
+      '   KoZIhvcNAQEBBQADggIPADCCAgoCggIBAK/9d/G5sOy4JQ6mqp/YG/jRo5KFytLe  '  +
+      '   K4i5sIPFviP9TixolyF1YrrVpOvYtJOI7mVcfsSuNvB4WRX4zMw559oLH1WOBfye  '  +
+      '   Tr3Y1t8uGNlb0v0nqN/DDpT9LyFm+rMw/AiA3XwInPPB80cZ6jOoABb2KX/MJj3D  '  +
+      '   4AS0osnW9uY91iYsyMXiCaEl84f/MMXQLGgRfK9PKLbWGIm0OJVScOtz3ZuppK9N  '  +
+      '   VIJWnhVpJZciVUhlb3b2uxM8oAz7WcuayUv1tlKrxSMnG3/Bhm7HfuF9jhmUoiOH  '  +
+      '   N7rOSCWlOL5YB9rz+I4kQFe/g2gOVWk1jPk1N8Fa+R57Hdcn12zCD91zVGCZH23x  '  +
+      '   2SirPHauhvetrsbNwMyQDXeA7pM5f3mLqavMGb55m8VHl1hA+nbwvIOEBwlSTvLu  '  +
+      '   9ajM7tS8rm8Pic2g5eJM2klArdkrvtdW6RAK33TbsqTKZl+/IxUppZah5XbAdljp  '  +
+      '   2Q6cFmpvcGSwVBfk2Av9XD8ggpZ9L6+ph2k22bTSGOLJrqhN4FE22w+FfokoZmeH  '  +
+      '   6x2LNIdpag4BS17UbAshRhEjA28vUuEjYgsiGR+KhNffdN71SLzdh51QkWqLjgp4  '  +
+      '   zk/IFy+1IbzdN9iG0uDeZ9Rs57QmXZZ3FYHjjX86BUOlEQjffujN1IwmW2bhuO3F  '  +
+      '   KqRUe82r0I5dAgMBAAGjfzB9MA4GA1UdDwEB/wQEAwIFoDAdBgNVHSUEFjAUBggr  '  +
+      '   BgEFBQcDAQYIKwYBBQUHAwIwDAYDVR0TAQH/BAIwADAdBgNVHQ4EFgQUROK+AQOb  '  +
+      '   xKImWwYqxlonqTVG5rMwHwYDVR0jBBgwFoAUWuBBURGHrNpoh8AAwwZc4nktif0w  '  +
+      '   DQYJKoZIhvcNAQENBQADggIBAK3rJFJ79/zog8PaDJumkB0myRkxi8etKQrFEbAn  '  +
+      '   zrPRtayNhoR7a6WYHQASxcYTo9wV7lyxvdHdwgL7+mcsAuwHzGsnBN3pvZXqrCv0  '  +
+      '   YO3nvCvc35P8dMPuN57rxth6z04ql/nSXSQa89v2viEK34bYs+Ai7c9hg2d6MQrN  '  +
+      '   s7NpD2j3FxJmT7aIEYt8p/xkr5GNq48APu2mqDll3ZJhSVKgNuz2kRkJopJFvS3s  '  +
+      '   ybpGShwGlScaY8TVukP9XLrfUG30G3/AcD0A9XHqOomWsPmcFLrzA4qi1XFcwTqe  '  +
+      '   +yIvL/y3c2HQjW1zQ8JaqV/bnCQIdAjJ5Qomqlm02jtNqKdLftsqkhcVQ1T0Y+/C  '  +
+      '   +j3jZybRPTXw8XXGug0/x31NqzjaPip5lXAOzHaf+8xqt3KOtKzV+12VCSnoaOIJ  '  +
+      '   x/a0J5unqVBdZlNWsg222q8njavUSQWIRoYxGpWHxmQoBDBD4bQK9JiOpBcsOXCu  '  +
+      '   3egjBowvyBWgTfY8LLXEX1ZSzY2mljaAsYuSeFGHepmAmXGPXU4TkfREzO062gAB  '  +
+      '   uGWmuzbxWbvz79zhlaWTbdLMpW6VdgQty3Ua9LEOuKzK2dSsPrd4YEXSc4AywuqJ  '  +
+      '   5sdnzFa4dJKrDcBws5JmetneJpaote4Hqicj5t4leOutIFFwb7J/8xDzpc9qgFvE  '  +
+      '   eyB5  '  +
+      '   -----END CERTIFICATE-----  '  +
+      '    ' ;
+    let documentJson = JSON.parse(qrString);
+    let claimWithLink = Object.values(documentJson)[0][0];
+    let claimData = Object.values(claimWithLink)[0];
+    let tempdata = claimData[0];
+    let moredata = Object.values(tempdata)[0];
+    console.log(moredata);
+    paperWallet.getCore().registerConnector('ephemeral', new EphemeralConnector())
+    let attestorSsid = await (await paperWallet.getCore().getConnector('ephemeral')).newIdentity({'cert': cert})
+    let validatorSsid = await paperWallet.getCore().newSsid('ephemeral')
```

Comment:
The way we use the paperWallet, we do not actually need to pass a validatorDid. This is only needed if you then want to read the data from the ephemeral connector. We actually just use the data we extracted from the QR.

I would not generate the validatorSsid, and just remove that argument in the validate call